top-app-bar/Flutter/Flutter-top-app-bar.md

```diff
@@ -21,7 +21,9 @@ There are two types of top app bar:
 1. [Contextual action bar](#contextual-top-app-bar)
 <br>
 
-![Image of regular and contextual app bars](Top-app-bars_types_side-by-side.png)
+![Regular app bar: purple background, white text and icons](assets/regular_app_bar_example.png)
```

Comment:
Is there a reason why you're using the separate images instead of the composite?

Reply:
Done